src/Logic.hs

```diff
@@ -251,16 +251,19 @@ handlePullRequestClosed pr state = return $ Pr.deletePullRequest pr state {
 isMergeCommand :: TriggerConfiguration -> Text -> Bool
 isMergeCommand config message =
   let
-    messageCaseFold = Text.toCaseFold $ Text.strip message
+    messageCaseFold = Text.words $ Text.toCaseFold $ Text.strip message
     prefixCaseFold = Text.toCaseFold $ Config.commentPrefix config
+    -- Recursive function that checks a list of words, if the first element is
+    -- the bot prefix and the second merge that we have a merge command. otherwise
+    -- we recurse to check the rest of the message.
+    isMergeCommand' :: Text -> [Text] -> Bool
+    isMergeCommand' prefix messageWords =
+      case messageWords of
+        (p:c:_) | p == prefix && c == "merge" -> True
+        (_:xs)  -> isMergeCommand' prefix xs
+        _       -> False
```

Comment:
So apart from space differences, this is similar to

```haskell
(prefixCaseFold <> " merge") `Text.isInfixOf` messageCaseFold
```

If you don't care about space, then I think the `isInfixOf` is a bit cleaner (and probably faster too). If you do care about the space, then we could use `List.isInfixOf`:

```haskell
[prefixCaseFold, "merge"] `List.isInfixOf` messageWords
```

Reply:
I will change this, your suggestion is much cleaner. I guess I should have [hoogle'd](https://hoogle.haskell.org/?hoogle=%5Ba%5D+-%3E+%5Ba%5D+-%3E+Bool&scope=set%3Astackage) before trying to roll my own solution